framewirc/utils.py

```diff
@@ -45,17 +45,28 @@ def __init__(self, **kwargs):
             raise exceptions.MissingAttributes(missing_attrs)
 
 
-def to_unicode(bytestring):
-    """Try to decode as UTF8, then fall back to cchardet."""
+def to_unicode(bytestring, *encodings):
+    """Try to convert a string of bytes into a unicode string."""
     # If we already have a unicode string, just return it.
     if isinstance(bytestring, str):
         return bytestring
 
-    try:
-        return bytestring.decode()
-    except UnicodeDecodeError:
-        charset = cchardet.detect(bytestring)['encoding'] or 'utf-8'
-        return bytestring.decode(charset, 'surrogateescape')
+    # If no encodings were passed through, give utf8 a go.
+    if not encodings:
+        encodings = ('utf8',)
```

Comment:
Is it possible to do `def to_unicode(bytestring, *encodings=('utf8',)):`, I wonder?

I suppose I ought to give it a go...